packaging/ubuntu-16.04/control

```diff
@@ -64,6 +64,7 @@ Depends: adduser,
          systemd,
          ${misc:Depends},
          ${shlibs:Depends}
+Recommends: squashfuse
```

Comment:
this is identical to https://github.com/snapcore/snapd/pull/2856 which was closed because this breaks "apt-get upgrade" ... was there an fix to this issue in some other place of the system ?